services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -64,6 +65,20 @@ def validate_user_status(*, user: dict, support_email: str):
     assert user_status == ACTIVE  # nosec
 
 
+async def notify_user_confirmation(
+    app: web.Application,
+    user_id: UserID,
+    product_name: ProductName,
+):
+    """Broadcast that user with 'user_id' has login for the first-time in 'product_name'"""
+    await observer.emit(
```

Comment:
Just a question for the decision behind this: Why didn't you go for rabbitmq (pub/sub pattern) since we have a distributed system and these events could be captured by multiple parts of the system?

Reply:
@GitHK Very good point. I totally agree with you. It was my original idea but I adopted a simpler solution until i introduce product in the invitations. I should create an issue so it does not remain like that forever though.